src/lang/english.txt

```diff
@@ -1041,6 +1041,14 @@ STR_GAME_OPTIONS_GUI_SCALE_3X                                   :3x
 STR_GAME_OPTIONS_GUI_SCALE_4X                                   :4x
 STR_GAME_OPTIONS_GUI_SCALE_5X                                   :5x
 
+STR_GAME_OPTIONS_PARTICIPATE_SURVEY_FRAME                       :{BLACK}Automated survey
+STR_GAME_OPTIONS_PARTICIPATE_SURVEY                             :{BLACK}Participate in automated survey
+STR_GAME_OPTIONS_PARTICIPATE_SURVEY_TOOLTIP                     :{BLACK}When enabled, OpenTTD will transmit a survey when leaving a game
+STR_GAME_OPTIONS_PARTICIPATE_SURVEY_LINK                        :{BLACK}About survey and privacy
+STR_GAME_OPTIONS_PARTICIPATE_SURVEY_LINK_TOOLTIP                :{BLACK}This opens a browser to https://survey.openttd.org/privacy with more information about the automated survey
```

Comment:
```suggestion
STR_GAME_OPTIONS_PARTICIPATE_SURVEY_LINK_TOOLTIP                :{BLACK}This opens a browser to {RAW_STRING} with more information about the automated survey
```
and fill with `NETWORK_SURVEY_DETAILS_LINK` (but I'm not sure it's possible for tooltips with the window system)

Anyway the URI is wrong here.

Reply:
Seems the only way to handle that is `OnTooltip()`

Reply:
The URL should never been in here. It isn't helpful (as you can't copy it) anyway :P So I just removed the URL :)

Reply:
Even simpler :)